### PR TITLE
Config-tools: generate ACFG_MAX_PCI_BUS_NUM based on board.xml

### PR DESCRIPTION
**Pull request by yuchuyang**

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: misc/config_tools/static_allocators/board_capability.py
Comment (junjiemao1) [inline misc/config_tools/static_allocators/board_capability.py:12]:
```suggestion
    common.append_node("/acrn-config/platform/MAX_PCI_BUS_NUM", hex(max(map(lambda x: int(x, 16), pci_bus_nums)) + 1), allocation_etree)
```

We'd better compare bus numbers as numbers. Otherwise the result could be wrong if both bus "0xf" and "0x10" are present.

File: misc/config_tools/xforms/config_common.xsl
Comment (junjiemao1) [inline misc/config_tools/xforms/config_common.xsl:323]: The indentation of the added lines looks strange.

**Review by junjiemao1 — CHANGES_REQUESTED** [review]

**Comment by junjiemao1** [comment]
Fine with me for the config tools part.

**Comment by yuchuyang** [comment]
@NanlinXie The configuration tool part is reviewed. Shall we merge it or wait for hypervisor maintainer's approval?@YuanyuanZhao609 Do you have any concern of this change?

**Comment by wenlingz** [comment]
@dongyaozu please help approve this PR, thanks

**Review by NanlinXie — APPROVED** [review]
Approve for configtool part.

**Pull request merged**
